raster/src/main/scala/geotrellis/raster/GridExtent.scala

```diff
@@ -17,10 +17,14 @@
 package geotrellis.raster
 
 import geotrellis.vector.{Extent, Point}
+import geotrellis.util.Constants
 
-import scala.math.{min, max, ceil}
 import spire.math.{Integral}
 import spire.implicits._
+import cats.SemigroupK
+import cats.implicits._
```

Comment:
` import cats.syntax.semigroup._` would be better for library purposes, and improves the compilation time.